.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+<!--  BUG REPORT -->
+<!--- Make sure these boxes are checked before submitting issue -->
+<!--  If this is NOT a bug report, you can delete this checklist. -->
+
+- [ ] I comply with the [Prerequisities](https://github.com/CodeSpaceHQ/MENGEL#prerequisities) and
+everything is up to date.
+- [ ] I have correctly followed all usage documentation.
+
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.  
+2.  
+3.  
+4.  
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
```

Comment:
https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#filing-bugs

@asclines sorry about the delay